controllers/apps/apimanager_controller.go

```diff
@@ -470,7 +470,7 @@ func (r *APIManagerReconciler) reconcileAPIManagerLogic(cr *appsv1alpha1.APIMana
 		return result, err
 	}
 
-	zyncReconciler := operator.NewZyncReconciler(baseAPIManagerLogicReconciler)
+	zyncReconciler := operator.NewZyncReconciler(baseAPIManagerLogicReconciler, cr.IsZyncEnabled())
```

Comment:
Instead of doing that, would it not be better to use the GetZyncOptions func? Not a big deal

Reply:
I think the current approach `cr.IsZyncEnabled()` is less computationally expensive because to use `GetZyncOptions()` you first need to create a NewZyncOptionsProvider which we then do again when creating the reconciler right after. So I think I'll leave as is if it's all the same.